budgetr-backend/src/main/java/pl/mantiscrab/budgetr/domain/user/User.java

```diff
@@ -2,16 +2,21 @@
 
 import lombok.*;
 
-import javax.persistence.*;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
 import javax.validation.constraints.Email;
 import javax.validation.constraints.NotBlank;
 
-@AllArgsConstructor
-@NoArgsConstructor
-@Getter
+@AllArgsConstructor(access = AccessLevel.PACKAGE)
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Builder()
+@Getter(AccessLevel.PACKAGE)
+@EqualsAndHashCode
 @Entity
-@Table(name = "budgetr_users")
-class User {
+@Table(name = "BUDGETR_USERS")
+public class User {
```

Comment:
I see you decided to make compromise :-D Sometimes it's worth making the interface as the api.

Reply:
You are right, but I didn't won't to mess up with JPA for now and decided for this solution